src/backend/joanie/edx_imports/api/__init__.py

```diff
@@ -0,0 +1,45 @@
+"""Viewset for course_run in the edx_imports app."""
+
+from http import HTTPStatus
+
+from rest_framework.decorators import (
+    api_view,
+    authentication_classes,
+    permission_classes,
+)
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from joanie.core.models import CourseRun
+from joanie.core.permissions import HasAPIKey
+from joanie.core.serializers import AdminCourseRunSerializer
+
+
+@api_view(["GET"])
+@permission_classes([HasAPIKey])
+@authentication_classes([])
+def course_run_view(request: Request):
+    """
+    Look for course_runs filter by a resource_link.
+    """
+    resource_link = request.query_params.get("resource_link")
+    if resource_link is None:
+        return Response(
```

Comment:
```suggestion
        return JsonResponse(
```
Would it be better since we return only a simple dictionary ? WDYT ?

Reply:
The response from drf is able to change the content-type based on the data. The response sent has the `Content-Type` with `application/json`, so there is no need to mix multiple response.